src/Tmds.Ssh/SshConnection.cs

```diff
@@ -0,0 +1,40 @@
+﻿// This file is part of Tmds.Ssh which is released under LGPL-3.0.
+// See file LICENSE for full license details.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+
+namespace Tmds.Ssh
+{
+    public sealed class SshConnection : IDisposable
```

Comment:
Just to confirm, we're only going to support ssh v2? No backwards 1.x support, right?

Reply:
Yes, only things which are 'recent'.